spec/integration/models/project_spec.rb

```diff
@@ -160,65 +160,67 @@
     end
   end
 
-  describe '#export_clone' do
-    context 'when exclude_schedule is true' do
-      let(:options) { { exclude_schedules: true } }
-      let(:clone) { GoodData::Project.create(title: 'project clone', client: @client, auth_token: ConnectionHelper::GD_PROJECT_TOKEN) }
-
-      after do
-        clone.delete if clone
+  describe 'cloning' do
+    let(:clone) { GoodData::Project.create(title: 'project clone', client: @client, auth_token: ConnectionHelper::SECRETS[:gd_project_token]) }
```

Comment:
warning: Similar code found in project_spec.rb:164, project_spec.rb:208 (mass = 32) (flay)